subprojects/docs/src/docs/userguide/dependencyManagement.adoc

```diff
@@ -47,156 +47,151 @@ It is ironic that in a language known for its rich library of open source compon
 
-[[sub:versioning_the_jar_name]]
-==== Put the Version in the Filename (Version the jar)
++++++
+<figure xmlns:xi="http://www.w3.org/2001/XInclude">
+    <title>Resolving binary dependencies from remote repositories</title>
+    <imageobject>
+        <imagedata fileref="img/dependency-management-binary-dependencies.png" width="150mm" />
+    </imageobject>
+</figure>
++++++
 
-The version of a library must be part of the filename. While the version of a jar is usually in the Manifest file, it isn't readily apparent when you are inspecting a project. If someone asks you to look at a collection of 20 jar files, which would you prefer? A collection of files with names like `commons-beanutils-1.3.jar` or a collection of files with names like `spring.jar`? If dependencies have file names with version numbers you can quickly identify the versions of your dependencies.
+===== Declaring a concrete version of a binary dependency
 
-If versions are unclear you can introduce subtle bugs which are very hard to find. For example there might be a project which uses Hibernate 2.5. Think about a developer who decides to install version 3.0.5 of Hibernate on her machine to fix a critical security bug but forgets to notify others in the team of this change. She may address the security bug successfully, but she also may have introduced subtle bugs into a codebase that was using a now-deprecated feature from Hibernate. Weeks later there is an exception on the integration machine which can't be reproduced on anyone's machine. Multiple developers then spend days on this issue only finally realising that the error would have been easy to uncover if they knew that Hibernate had been upgraded from 2.5 to 3.0.5.
-
-Versions in jar names increase the expressiveness of your project and make them easier to maintain. This practice also reduces the potential for error.
+The code snippet below declares a compile-time dependency on the Spring web module by its coordinates: `org.springframework:spring-web:5.0.2.RELEASE`. Gradle resolves the dependency including its transitive dependencies at runtime from the link:https://search.maven.org/[Maven Central repository] and uses it to compile Java source code. The version attribute of the dependency coordinates points to a _concrete version_ indicating that the underlying artifacts don't change over time. The use of concrete versions ensures reproducibility for the aspect of dependency resolution.
```

Comment:
I think Maven Central is still far more popular than Bintray. Readers will much easier find themselves at home if we refer to something that has been established for over a decade. Why would you prefer Bintray here?

Reply:
We generally recommend Bintray as a _strictly better_ alternative: it's faster and mirrors Maven Central, not to mention isn't associated with a competitor.

Reply:
I personally wouldn't see Maven Central as a competitor. It's just a repository and not the build tool per se. Thinking back to users migrating from a Maven build I'd think that they'd rather understand what's going here if we'd use Maven Central instead of something they never heard of.